app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">ERPA</string>
+    <string name="launch_game_list_button">Game List</string>
+    <string name="launch_create_game_button">Create Game</string>
```

Comment:
We agreed not to worry about internationalization features, can you please remove this resources and use an explicit value on your activities?